addons/l10n_pe/models/account_journal.py

```diff
@@ -0,0 +1,27 @@
+from odoo import models, api
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    @api.model
+    def _get_sequence_prefix(self, code, refund=False):
+        """For peruvian companies we can not use sequences with **/** due to the edi generation which need in the
+        sequence a plain text ended by a *-* and the length of this prefix"""
+        if self.env.company.country_id != self.env.ref('base.pe'):
+            return super()._get_sequence_prefix(code, refund=refund)
+        prefix = code.upper()
```

Comment:
So, the standard shortcode is ok as a prefix?  And why do we remove one char for the refund?  I tested this in runbot and in my sequence I have no prefix at all.

Reply:
We must have 3 chars followed by a dash (-), the removal is only referential for Credit note.

do this answer the question?